.github/workflows/publish-release-container.yml

```diff
@@ -0,0 +1,42 @@
+name: Build and Publish container for new releases
+
+on: [push] # debug
```

Comment:
Does this build and publish the image when I create a release through github ui?

Reply:
yup, the commented lines below do exactly this